src/rbus/dsm_rbus_provider.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef DSM_RBUS_PROVIDER_H
```

Comment:
Missing header on new file

Reply:
We have an RBUS integration with DSM which will eventually be made public, but not in this pull request which is about internal fixes for DSM. We'll review the RBUS integration code when we release it.